setup/setup.sh

```diff
@@ -2,8 +2,25 @@
 
 
 # Global Variables
+arch="$(uname -m)"
 runuser="$(whoami)"
+trueuser="$(who am i | awk '{print $1}')" # if this is blank, we're actually root (kali)
```

Comment:
Instead of ```$(who am i | awk '{print $1)```, we can simply call ```$(whoami)``` and it will return only the username. No need to use awk.